### ISSUE TITLE
Dancerfly default waiver unenforceable and unethical?

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#875]
c/w for mentions of violence/sexual violence.

I got some pretty serious negative feedback from a lawyer on the default dancerfly waiver:

> I'd like to register, but I'm not willing to sign a document that says I will release [event]'s officers from "all liability of injury, loss, or damage to personal property" for the event.  I do not believe that is enforceable as written, and rightfully so; it is not reasonable to release [event]'s officers from liability for intentional torts.
>
> I mean, seriously, if one of you punches somebody or rapes somebody at [event], you really want the perpetrator not to be legally liable for that?
>
> I don't understand the goals of the waiver you have-- if you want it to actually be a release of claims, it's almost certainly unenforceable.  If you want it to create an expectation that people will not sue any of you or [event] for anything, no matter how horrible, why is that an expectation you are trying to create?  I just don't understand why you would want people to think that this is an event where the organizers don't intend to take responsibility for their own behavior.  I go to a decent number of dance weekends and I have literally never been asked to sign anything this broad, and (in case this wasn't obvious), being asked to do so offends me.

His suggestion for this particular event was to not have a waiver at all saying, "Even if you were to write an enforceable waiver, the benefits of it are really quite small," and suggesting maybe we should replace it with someone that requested adherence to our code of conduct instead.

Solutions:

- Minimally I think we should update the waiver to something closer to legally enforceable and less broad.
- Maybe we should alter that field to be a general "Agreement" field. We could even include a default CoC/Safer Space policy!
- This would also be partially mitigated by resolving #427

**Comment by melinath** [comment]
Waivers used by dance events have basically always been less enforceable than the paper they were written on. That being said, if there actually were a legally useful waiver we could use, that would be preferable.

**Comment by harrislapiroff** [comment]
Yeah, I'm most concerned with the ethical aspects of what they said. I agree that it's pretty ethically suspect to expect people to sign away their right to hold organizers or an event liable for _literally anything_, even if this is a common practice. It's not the worst problem since this field is and always has been customizable, but we should be promoting best practices with the Dancerfly defaults, not just the standard ones. Which is why I particularly like the idea of making it an agreement field and including a CoC or a reference to a CoC in it.
